src/wwwroot/app/services/VacancyService.js

```diff
@@ -45,6 +47,13 @@ export default class VacancyService {
       _UserService = UserService;
    }
 
+   getVacancy(id) {
+      let additionalUrl = VACANCY_URL + id;
+      return _HttpService.get(additionalUrl).then((vacancy) =>
```

Comment:
I wrote this line
`return _HttpService.get(additionalUrl).then(this.convertFromServerFormat);`

why you don't use it?